README.md

```diff
@@ -0,0 +1,75 @@
+# clay-sitemaps
+
+A suite a tools for generating sitemaps for Clay sites.
+
+Clay-sitemaps provides two "standard" sitemap middleware, one for text and one for XML, that can get you up and running quickly. It also provides utilities that can help you create your own sitemap logic.
+
+## Quick Start
+
+```
+const sitemaps = require('clay-sitemaps')(someAmphoraInstance);
+
+app
+  .get('/sitemap.txt', sitemaps.standardText())
+  .get('/sitemap.xml', sitemaps.standardXML({
+    engines: { // engines for rendering sitemap templates
+      handlebars: yourHandlebarsEngine
```

Comment:
@larrychevres Yeah, the `use`-able middleware approach would be more "plug and play." My thought was that the consumer might want to set the routes explicitly. But I don't imagine many will deviate from `/sitemap.xml` and `/sitemap.txt`, so I'll refactor.

Irrespective of that, should I expose `amphora`'s engine so consumers don't have to pass it explicitly to plugins like this?

Re: news sitemap — All URLs in any news sitemap needs two things:`<title>` and `<publication_date>`. As far as I understand, none of these pieces of info really belong to a page object; instead, they can be derived from the data of a page's components. So we _could_ offer a "batteries included" solution for that, but it would be opinionated about the components pages should include and where.

The approach I thought we might take instead is to expose the various pieces of `clay-sitemaps` so that one could use them a la carte to craft custom sitemap logic, and do that ourselves to make our news sitemap. Here's how that would look in `sites`. `newsTransforms` includes some stream filter / transform functions specific to our requirements.

```
app.get('/news.xml', (req, res)=>{
      sitemaps.streamPages(res.locals.site.prefix)
        .pipe(sitemaps.filters.published())
        .pipe(sitemaps.filters.public())
        .pipe(sitemaps.transforms.pages.compose(res.locals))
        .pipe(newsTransforms.filterMaxAge(2))
        .pipe(newsTransforms.filterMinWordCount(100))
        .pipe(newsTransforms.pagesToNewsXML(locals.site.name, ['someTags', 'toIgnore']))
        .pipe(sitemaps.transforms.strings.addBookends(NEWS_XML_PRELUDE, '</urlset>'))
        .pipe(res)
```